release of fremen2dgrid failed: missing dep in package.xml

https://lcas.lincoln.ac.uk/jenkins/job/ros-indigo-fremen2dgrid_binarydeb_trusty_amd64/1/console

```
CMake Error at /opt/ros/indigo/share/catkin/cmake/catkinConfig.cmake:75 (find_package):
  Could not find a package configuration file provided by "nav_msgs" with any
  of the following names:

    nav_msgsConfig.cmake
    nav_msgs-config.cmake

  Add the installation prefix of "nav_msgs" to CMAKE_PREFIX_PATH or set
  "nav_msgs_DIR" to a directory containing one of the above files.  If
  "nav_msgs" provides a separate development package or SDK, be sure it has
  been installed.

```

--- a/Fremen2DGrid/package.xml
+++ b/Fremen2DGrid/package.xml
@@ -40,9 +40,12 @@
   <build_depend>roscpp</build_depend>
   <build_depend>genmsg</build_depend>
   <build_depend>message_runtime</build_depend>
+  <build_depend>nav_msgs</build_depend>
+  
   <run_depend>roscpp</run_depend>
   <run_depend>message_runtime</run_depend>
   <run_depend>genmsg</run_depend>
+  <run_depend>nav_msgs</run_depend>
 
   <build_depend>actionlib</build_depend>
   <build_depend>actionlib_msgs</build_depend>

